commands/dkcheck.ml

```diff
@@ -122,12 +117,15 @@ let _ =
   let export       = ref false in
   let beautify     = ref false in
   let options = Arg.align
-    [ ( "-v"
-      , Arg.Set verbose
-      , " Enable the verbose mode" )
-    ; ( "-d"
+    [ ( "-d"
       , Arg.Int Basic.set_debug_mode
       , "N sets the debuging level to N" )
```

Comment:
Ceci n'est pas en accord avec ton commentaire dans Basic.mli . Peut-être que ce n'est pas du debug, mais plus des informations.

Reply:
Changed the comment to "verbosity" level.